src/main/java/Deadline.java

```diff
@@ -0,0 +1,57 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Task called Deadline.
+ * A deadline has a attribute "dueDateAsString" or "dueDateAsDate", which is the time the deadline is due.
+ * The attribute dueDate can be represented by a String, or a date object.
+ */
+public class Deadline extends Task{
+    protected String dueDateAsString;
+    protected LocalDate dueDateAsDate;
+
+    Deadline(String taskDescription, String date) {
+        this.taskDescription = taskDescription;
+        this.isCompleted = false;
+        this.dueDateAsString = date;
+        this.dueDateAsDate = null;
+    }
+
+    Deadline(String taskDescription, LocalDate date) {
+        this.taskDescription = taskDescription;
+        this.isCompleted = false;
+        this.dueDateAsDate = date;
+        this.dueDateAsString = null;
+    }
+
+    /**
+     * Return the due date of the deadline.
```

Comment:
Should it be `Returns` rather than `Return`? I noticed the same issue in several other places as well.